src/libs/actions/App.js

```diff
@@ -105,46 +101,54 @@ AppState.addEventListener('change', (nextAppState) => {
 
 /**
  * Fetches data needed for app initialization
- * @returns {Promise}
  */
 function getAppData() {
     BankAccounts.fetchUserWallet();
-
-    // We should update the syncing indicator when personal details and reports are both done fetching.
-    return Promise.all([
-        Report.fetchAllReports(true),
```

Comment:
Coming from https://github.com/Expensify/App/issues/32658

Not loading all reports broke the IOU report previews.